frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Media, Button } from 'reactstrap';
+import { Text } from 'react';
+import './Features.css'
+import spacePic from './spacepic.jpg';
+import musicPic from './musicpic.jpg';
+import newsPic from './newspic.jpg';
+import weatherPic from './weatherpic.jpg';
+import factPic from './factpic.jpg';
```

Comment:
What do you think about [moving](https://github.com/Codubee/SpaceJunkTTH2-4/tree/main/frontend/public) the pictures to be inside of the public folder and [importing them](https://github.com/Codubee/SpaceJunkTTH2-4/blob/ad2164e09734a905401f879e312de3ecca85152d/frontend/src/App.js#L1) like this?